src/PeopleSDKAdapter.test.js

```diff
@@ -0,0 +1,111 @@
+
+    mockSDKPerson = {
+      id: 'id',
+      emails: ['email@cisco.com'],
+      displayName: 'Webex Components',
+      nickName: 'Teams',
+      firstName: 'Webex',
+      lastName: 'Components',
+      avatar: 'avatar',
+      orgId: 'orgID',
+      created: 'created',
+      type: 'person',
+    };
+    personID = 'personID';
+  });
+
+  test('fetchPerson() fetches person details properly', async () => {
+    mockSDK.people.get = jest.fn(() => Promise.resolve(mockSDKPerson));
+    const personResponse = {
+      ID: mockSDKPerson.id,
+      emails: mockSDKPerson.emails,
+      displayName: mockSDKPerson.displayName,
+      firstName: mockSDKPerson.firstName,
+      lastName: mockSDKPerson.lastName,
+      avatar: mockSDKPerson.avatar,
+      orgID: mockSDKPerson.orgId,
+    };
+
+    expect(await peopleSDKAdapter.fetchPerson(personID)).toEqual(personResponse);
+  });
+
+  describe('getPerson()', () => {
+    test('returns an observable', () => {
+      peopleSDKAdapter.fetchPerson = jest.fn(() => Promise.resolve(mockSDKPerson));
+      mockSDK.internal.presence.subscribe = jest.fn(() => Promise.resolve({responses: [{status: {status: 'active'}}]}));
+
+      expect(isObservable(peopleSDKAdapter.getPerson(personID))).toBeTruthy();
+    });
+
+    test('returns a person in a proper shape', (done) => {
+      mockSDKPerson.status = 'ACTIVE';
+      peopleSDKAdapter.fetchPerson = jest.fn(() => Promise.resolve(mockSDKPerson));
+      mockSDK.internal.presence.subscribe = jest.fn(() => Promise.resolve({responses: [{status: {status: 'active'}}]}));
+
+      peopleSDKAdapter.getPerson(personID).subscribe((person) => {
+        expect(person).toEqual(mockSDKPerson);
+        done();
+      });
+    });
+
+    test('stops listening to events when unsubscribing', () => {
+      peopleSDKAdapter.fetchPerson = jest.fn(() => Promise.resolve(mockSDKPerson));
+      mockSDK.internal.presence.subscribe = jest.fn(() => Promise.resolve({responses: [{status: {status: 'active'}}]}));
+
+      const subscription = peopleSDKAdapter.getPerson(personID).subscribe();
+
+      subscription.unsubscribe();
+      expect(mockSDK.internal.presence.unsubscribe).toHaveBeenCalled();
+    });
+
+    test('throws a proper error message for an invalid person ID to fetch the person details from', (done) => {
+      const errorMessage = 'a proper people error message';
+
+      personID = 'invalid personID';
+      peopleSDKAdapter.fetchPerson = jest.fn(() => Promise.reject(new Error(errorMessage)));
+      mockSDK.internal.presence.subscribe = jest.fn(() => Promise.resolve({responses: [{status: {status: 'active'}}]}));
+
+      peopleSDKAdapter.getPerson(personID).subscribe(
+        () => {},
+        (error) => {
+          expect(error.message).toBe(errorMessage);
+          done();
+        }
+      );
+    });
+
+    test("throws a proper error message for an invalid person ID to subscribe to person's presence status update", (done) => {
```

Comment:
Not sure why this one use `"` instead of `'`
```suggestion
    test('throws error on failed presence update subscription', (done) => {
```